ignite/pkg/protoanalysis/builder.go

```diff
@@ -195,9 +196,9 @@ func (b builder) constantToHTTPRules(requestMessage *proto.Message, constant pro
 
 	// calculate url params, query params and body fields counts.
 	var (
-		messageFieldsCount = b.messageFieldsCount(requestMessage)
-		paramsCount        = len(params)
-		bodyFieldsCount    int
+		messageFields, messageFieldsCount = b.messageFieldsCount(requestMessage)
+		paramsCount                       = len(params)
+		bodyFieldsCount                   int
 	)
 
 	if body, ok := constant.Map["body"]; ok { // check if body is specified.
```

Comment:
`.Map` is deprecated, can you change to `.OrderedMap` instead

https://github.com/ignite/cli/issues/4721

Reply:
also here: https://github.com/ignite/cli/blob/b8c8fa17941fea487c85f001d45c382566d8391e/ignite/pkg/protoanalysis/builder.go#L173-L173

and here: https://github.com/ignite/cli/blob/b8c8fa17941fea487c85f001d45c382566d8391e/ignite/pkg/protoanalysis/builder.go#L253-L253